src/linear_algebra/lagrange.lean

```diff
@@ -77,26 +79,26 @@ begin
     { rw ← pow_one (X : polynomial F) at hz, exact X_pow_sub_C_ne_zero zero_lt_one _ hz } }
 end
 
-variables (f : s → F)
+variables (f : F → F)
```

Comment:
I guess one benefit of the `s → F` approach is that it would give a linear equiv between `s → F` and polynomials of degree less than `s.card`.
What do you think? Can we still state that easily?